test/topology_custom/test_read_repair.py

```diff
@@ -222,7 +222,7 @@ async def test_incremental_read_repair(data_class, workdir, manager):
 
     host1, host2 = await wait_for_cql_and_get_hosts(cql, [node1, node2], time.time() + 30)
 
-    cql.execute("CREATE KEYSPACE ks WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 2}")
+    cql.execute("CREATE KEYSPACE ks WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 2} AND tablets = { 'enabled': false }")
```

Comment:
Why did you change this line? If this test doesn't work on tablets because of a real, good, reason, please add a comment. If it doesn't work because on a bug I think you should add a comment referring to the open issue number - so we can later search for these tests when we finally fix the issue.

Reply:
I provided the comment.

Reply:
row_id, is_live = res
            if is_live:
                actual_live_rows.add(row_id)
            else:
                actual_dead_rows.add(row_id)

        # Account rows that have a tombstone but are live only once.
        actual_dead_rows -= actual_live_rows

>       assert actual_live_rows == expected_live_rows
E       AssertionError: assert {0, 48, 59, 88, 90} == {0, 4, 35, 83, 87, 93, ...}
E         Extra items in the left set:
E         48
E         59
E         90
E         88
E         Extra items in the right set:
E         99...
E
E         ...Full output truncated (6 lines hidden), use '-vv' to show

This is how the test fails (snippet from the log) without the change.